index.html

```diff
@@ -773,7 +773,7 @@
 			<li>If the pointer is not in the <a>active buttons state</a>, then terminate these steps.</li>
 			<li>For the specified <code>pointerId</code>, set the <dfn>pending pointer capture target override</dfn> to the <code>Element</code> on which this method was invoked.</li>
 		</ol>
-		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements.</div>
+		<div class="note">When pointer capture is set, <code>pointerover</code>, <code>pointerout</code>, <code>pointerenter</code>, and <code>pointerleave</code> events are only generated when crossing the boundary of the element that has capture as other elements can no longer be targeted by the pointer. This has the effect of suppressing these events on all other elements. The region of the capturing element includes the element itself and all its children to mimic the hover state of the capturing element. Moving out of (into) this region causes <code>pointerout</code> and <code>pointerleave</code> (<code>pointerover</code> and <code>pointerenter</code>) to be sent to the capturing element and all its ancestors.</div>
```

Comment:
Yup. That observation is correct too. I guess either we should go the second interpretation or if we go to the first one we should mention that capturing only suppresses the events on elements that are not in the ancestor tree of the capturing node.

Also I'd like to know what @jacobrossi and @teddink think and if they have a reason why Edge implements the first interpretation.